_targets.R

```diff
@@ -7,9 +7,36 @@ options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
 p1_targets_list <- list(
+  tar_target(
+    site_01427207,
+    download_nwis_data("01427207"),
```

Comment:
As mentioned in an earlier comment, I would recommend passing in the filepath as the argument here and then extracting the site number inside the function. 

I can't remember if this has been mentioned yet in the course (ignore if it hasn't) but when naming targets that are files, we have started following the convention of adding the file extension. So, a target that creates `data.csv` would be named `data_csv`. So in your case, it would be `site_01427207_csv` and so on.